### ISSUE TITLE
Exported map image resolution is dependent on current zoom level

### ISSUE THREAD

**Issue by Inglonias** [MegaMek/megamek#3426]
### Environment
MegaMek standalone 0.49.6

### Description
The resolution of an exported map image changes depending on how zoomed in your view in the map editor is.

### Files

**Zoomed out**
![zoomed_out](https://user-images.githubusercontent.com/2125926/152261818-b0dba786-f766-4c66-90b5-fae2c978545e.png)

**Zoomed in**
![zoomed_in](https://user-images.githubusercontent.com/2125926/152261822-8a1e9a7b-daf0-4650-baff-832ec66573c3.png)

**Comment by mjbroekman** [comment]
Oh wow. That is interesting. I was about to open an RFE for a similar situation (but with 0.48). At the default zoom level, when you export the image and print it, the hexes are too small for miniature bases to fit (both old-sculpt and new-sculpt bases). I ended up having to upscale the image between 115% and 125% depending on how tightly I wanted the bases to fit within the hexes:

- 115% => old-sculpt bases (25th anniversary boxed set minis) fit exactly
- 120% => new-sculpt bases (AGoAC / Clan Invasion boxed set minis) fit exactly on top of the hex edge lines (meaning mechs in adjacent hexes would have the bases collide)
- 125% => new-sculpt bases (AGoAC / Clan Invasion boxes set minis) fit _within_ the hex edges (meaning the *bases* won't collide, though the mechs may, depending on overhang)

The RFE(s) I was looking for was:
1. Allow scale (dpi / hex size) to be set at save time (for instance, hex = 1 inch, dpi = 72dpi would result in the _HEX_ being 1 inch WITHIN the hex lines). This would also allow for variation in line thickness (thin = 1 dot, thick = 2 dots, for instance).
2. Have a multi-page option in the "save as" to split the image in 'paper-size' sized images.
3. Have a "save as pdf" or "save as postscript" option to save directly to multi-page PDF/PS document to reduce the steps needed to take a MegaMek generated map and print it for tabletop use.

**Comment by Inglonias** [comment]
Yeah. I used MegaMek for Lancer RPG battlemaps, and I had a lot of trouble discovering the magic numbers I needed to use to align MegaMek's hex grids with that of Foundry VTT or Roll20. I was gobsmacked to learn that the zoom level actually changes the exported image.

**Comment by SimonLandmine** [comment]
Exported map resolution being related to current zoom level is a feature, not a bug, as it means that you can export the rendered image at the size you desire (and reduce memory usage, if required), without having to use separate graphics software to change the size of your battle screenshot.
An option to "Save map as image at maximum size" would be an RFE solution, however.
The megamek hexes were not originally intended to be printed out directly as a map, which is probably why the maximum zoomed-in size still does not output at mapsheet hex size, as the graphics are not created at that scale.
